content/_help/verify-your-identity/phone-number._fr.md

```diff
@@ -20,15 +20,18 @@ Nous pouvons plus facilement vérifier votre identité si vous saisissez votre n
 Vous pouvez utiliser un téléphone portable prépayé pour vérifier votre identité. Cependant, certains numéros de téléphone prépayés contiennent des facteurs de risque qui peuvent compromettre la vérification de votre identité.
 
 ## Vous ne pouvez pas vérifier votre identité avec :
-- Les numéros internationaux
+- Des numéros internationaux et des numéros de certains territoires américains
 - Google Voice ou des numéros VOIP (voix via le protocole internet) similaires
```

Comment:
And here it's capitalized incorrectly but translated to French.

Reply:
Let's also style this "VoIP" here.